lib/jekyll-admin.rb

```diff
@@ -32,6 +32,9 @@ def self.site
     @site ||= begin
       site = Jekyll.sites.first
       site.future = true
+      if site.config["baseurl"].nil?
+        site.config["baseurl"] = ""
```

Comment:
when `baseurl` is not in `_config.yml` I initialise it to `""` here, does someone know if it is a good way to go?